app/main.dev.js

```diff
@@ -257,12 +253,14 @@ app.on('ready', async () => {
 
   sendGrpcDisconnected()
   // Check to see if an LND process is running
-  lookup({ command: 'lnd' }, (err, results) => {
+  lookup({ command: 'node' }, (err, results) => {
```

Comment:
Why `node` here? The executable is `lnd`, `node` seems overly broad.

Reply:
Good catch @Empact - that was left over from when I was testing some thing locally, before I introduced the `ZAP_LND_DISABLE` flag. Have removed and updated the PR